appserver/security/ejb.security/src/main/java/com/sun/enterprise/iiop/security/InvalidMechanismException.java

```diff
@@ -17,13 +18,24 @@
 package com.sun.enterprise.iiop.security;
 
 public class InvalidMechanismException extends java.lang.Exception {
```

Comment:
It will need much more iterations, I will start enabling checkstyle rules soon. This PR is an excerpt from #24478 where I collided with usage of resources from different modules and decided that it is more simple+better to split it than trying to keep it consistent. Because as I go step by step, it makes these things visible.
So please don't ask to finish that all in a single step, important is direction ;)

Reply:
I will follow yet with another similar PR.